src/features/registry/metrics.ts

```diff
@@ -0,0 +1,13 @@
+import { metrics } from '@balena/node-metrics-gatherer';
+
+metrics.describe.counter(
+	'api_registry_image_pull_total',
+	'Number of image pulls getting the token via API',
+);
+
+export function incrementRegistryImagePulls(
+	value: number = 1,
```

Comment:
When you specify a default value then the type defaults to the type of that default value:
```suggestion
	value = 1,
```
There's nothing wrong with explicitly specifying `number` but it's redundant and makes sense imo to reserve for only the cases where you want a type that is different to that of the default value